### PR TITLE
Cart Page Added

### PR DESCRIPTION
**Pull request by aamrapaliwandhre**
chrome-capture (26)

### CODE REVIEW

**Review by KanekarSnehal — APPROVED** [review]
Looks good, only thing is your css code is not formatted properly.

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FinestStitch</title>
+    <link rel="stylesheet" href="distracted-jones-2f14b5.netlify.app/index.css" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (harshkotkar5557) [inline cart.html:10]: you can skip to add this font awsome link to every css file by just importing this in your main css file

**Review by MonikaJhingan — APPROVED** [review]
Code well indented looks good to merge.

**Review by Snigdha17 — APPROVED** [review]
UI looks good, you can format the code in cart.css so it is consistent

**Review by harshkotkar5557 — APPROVED** [review]
clean code, though I have given some suggestion

File: cart.html (hunk 1 above)
Comment (aamrapaliwandhre) [reply cart.html:10]: ok

**Review by Mahi24-prog — APPROVED** [review]
1. Remove  extra space in code and arrange properly. 
2. Try to make it responsive.

**Pull request merged**